### PR TITLE
Benny/netkes234 update logo

### PR DESCRIPTION
**Pull request by Bennyoak**
This PR includes:
- New Logos on all the things
- Change color scheme to orange
- New favicon.ico
- New loader gif
- Change login images to match color scheme

I'm not 1000% happy with the loader, but it'll do till I can come up with something better.

### CODE REVIEW

File: django/apps/blue_management/blue_mgnt/templates/base.html [hunk 1]
```diff
@@ -11,7 +11,7 @@
 	{% block base_styles %}
 	{% endblock base_styles %}
 
-    {% with dev=False %}
+    {% with dev=True %}
```
Comment (bdzim) [inline django/apps/blue_management/blue_mgnt/templates/base.html]: You forgot to change that back.

**Comment by bdzim** [comment]
It doesn't look like you built the css.

**Comment by Bennyoak** [comment]
Updated in https://github.com/SpiderOak/netkes/commit/4b22afefbfcc7aea150f7e7da5794ed77eaf293f

**Comment by bdzim** [comment]
Merged into 2.2.1

**Pull request closed without merging**